### PR TITLE
EMP-424 crm14 missing fields

### PR DESCRIPTION
**Pull request by solomonmanik**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LASB-XXX)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java [hunk 1]
```diff
@@ -609,6 +626,41 @@ default String assignFileKey(Crm14AttachmentModel processAttachModel) {
         }
         return null;
     }
+    @Named("hasCrm15")
+    default boolean hasCrm15(Crm14DetailsModel model) {
```
Comment (tanovellino) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java:630]: I think this function should be moved to class Crm14DetailsModel. That will encapsulate that logic which relies only on the model object itself. It can also make testing far easier. Besides, in my opinion, knowing if it has a 15 or not is not part of the mapper but of the model.
Happy to do this in another commit. But add a TOTO so we do not forget to do it at some point

Comment (solomonmanik) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java]: Thanks @tanovellino  Moved Mapper code to Model class.

**Review by tanovellino — APPROVED** [review]
LGM
